### PR TITLE
feat(docs): added links to oauth github tokens and nx cloud

### PR DESCRIPTION
**Pull request by antoniobfm**
A small improvement that can make the setup a little faster

### CODE REVIEW

**Review by Johnmiicheal — COMMENTED** [review]
👌

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -124,9 +124,9 @@ You will need to create a postgres database and get the URL of it to use as the
 
 We also use [Upstash](https://upstash.com/) for rate limiting and a redis instance.
 
-You will also need to configure oauth github tokens to be able to use the authentication functionality of Noodle.
+You will also need to configure [oauth github tokens](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens) to be able to use the authentication functionality of Noodle.
```
Comment (ajitzero) [inline CONTRIBUTING.md:127]:
```suggestion
You will also need to configure [GitHub OAuth tokens](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens) to be able to use the authentication functionality of Noodle.
```

Hi, could also please fix these minor typos?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -124,9 +124,9 @@ You will need to create a postgres database and get the URL of it to use as the
...
 
-Also, if you want your own personal nx cloud instance, you will need to create a project on nx cloud and replace the default read only token provided.
+Also, if you want your own personal [nx cloud](https://cloud.nx.app/) instance, you will need to create a project on nx cloud and replace the default read only token provided.
```
Comment (ajitzero) [inline CONTRIBUTING.md:129]:
```suggestion
Also, if you want your own personal [Nx Cloud](https://cloud.nx.app/) instance, you will need to create a project on Nx Cloud and replace the default read-only token provided.
```
And this as well.

**Review by ajitzero — APPROVED** [review]

**Review by ixahmedxi — CHANGES_REQUESTED** [review]
We have moved to clerk for authentication now which doesn't need github oauth tokens unless you want to move it to production which you can't under our license (unless it's us). 

So I would like to have some kind of instruction about setting up clerk's environment variables.

**Comment by antoniobfm** [comment]
> We have moved to clerk for authentication now which doesn't need github oauth tokens unless you want to move it to production which you can't under our license (unless it's us).
> 
> So I would like to have some kind of instruction about setting up clerk's environment variables.

Ok, I will take a look into that!

**Review by antoniobfm — COMMENTED** [review]
I've modified the document to include clerk instead of github oauth tokens and a little tutorial on how to configure it.

What do you think?

**Review by ixahmedxi — APPROVED** [review]

**Pull request merged**